package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "zen-site-next",
-  "packageManager": "yarn@4.1.0",
+  "packageManager": "pnpm@9.0.6+sha256.0624e30eff866cdeb363b15061bdb7fd9425b17bc1bb42c22f5f4efdea21f6b3",
   "scripts": {
     "dev": "next dev",
+    "dev:debug": "NODE_OPTIONS='--inspect' next dev",
```

Comment:
In order to debug in vscode, this needs to be set to the following:

```NODE_OPTIONS='--inspect=0.0.0.0' next dev```

However, the following error occurs, which is documented in https://github.com/vercel/next.js/issues/53127, which still does not work even when upgrading next. This issue will be punted to another task. 

```
➜  zen-site-next git:(dockerize) docker compose up --build
[+] Building 87.7s (12/12) FINISHED                                                                                            docker:desktop-linux
 => [app internal] load build definition from Dockerfile                                                                                       0.0s
 => => transferring dockerfile: 282B                                                                                                           0.0s
 => [app internal] load metadata for docker.io/library/node:18-alpine                                                                          0.6s
 => [app internal] load .dockerignore                                                                                                          0.0s
 => => transferring context: 216B                                                                                                              0.0s
 => [app 1/7] FROM docker.io/library/node:18-alpine@sha256:4837c2ac8998cf172f5892fb45f229c328e4824c43c8506f8ba9c7996d702430                    0.0s
 => [app internal] load build context                                                                                                          0.8s
 => => transferring context: 72.31MB                                                                                                           0.8s
 => CACHED [app 2/7] WORKDIR /app                                                                                                              0.0s
 => [app 3/7] COPY package.json ./                                                                                                             0.1s
 => [app 4/7] COPY pnpm-lock.yaml ./                                                                                                           0.0s
 => [app 5/7] RUN corepack enable                                                                                                              0.6s
 => [app 6/7] RUN pnpm install                                                                                                                28.8s
 => [app 7/7] COPY . .                                                                                                                        11.2s
 => [app] exporting to image                                                                                                                  35.9s
 => => exporting layers                                                                                                                       35.2s
 => => writing image sha256:0d264e9dc620645580496252b56b4f5d23e23f2bd904f8ebe59c5a5804ba57c9                                                   0.2s
 => => naming to docker.io/library/zen-site-next-app                                                                                           0.0s
[+] Running 1/1
 ✔ Container zen-site-next-app-1  Recreated                                                                                                    0.2s
Attaching to app-1
app-1  |
app-1  | > zen-site-next@ dev:debug /app
app-1  | > NODE_OPTIONS='--inspect=0.0.0.0' next dev
app-1  |
app-1  | Debugger listening on ws://0.0.0.0:9229/d4d2bd4a-6dbd-49ce-afa4-34248380d560
app-1  | For help, see: https://nodejs.org/en/docs/inspector
app-1  | /usr/local/bin/node:  must be 0 or in range 1024 to 65535.
app-1  |
app-1 exited with code 0
```